x/bank/migrations/v043/store_test.go

```diff
@@ -57,6 +58,10 @@ func TestSupplyMigration(t *testing.T) {
 		Amount: amount,
 	}
 	require.Equal(t, oldBarCoin, newBarCoin)
+
+	// foobar shouldn't be existed in the store.
+	bz = supplyStore.Get([]byte("foobar"))
+	require.Nil(t, bz)
 }
 
 func TestBalanceKeysMigration(t *testing.T) {
```

Comment:
Can we add a test for pruning zeros balances too?

It can be inside `TestBalanceKeysMigration` or it can be in another test.